recipes/etc-local.rb

```diff
@@ -30,6 +30,11 @@
       config_path = site['docroot']
     end
 
+    directory "#{config_path}/app/etc" do
+      recursive true
+      action :create
+    end
+
```

Comment:
doh, doesn't look like chef made that possible, I guess the alternative is a `not_if`